include/psp2/pss.h

```diff
@@ -12,7 +12,17 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+  
+typedef struct PSM_handle {
```

Comment:
To be consistent, I'd change the name to something like `ScePssHandle`, or maybe `ScePssCryptoHandle` if this handle is only used for PSS crypto functions.